### ISSUE TITLE
This extension may soon no longer be supported because it doesn't follow best practices for Chrome extensions.

### ISSUE THREAD

**Issue by Imran-imtiaz48** [tulios/json-viewer#361]

**Comment by dev-enoch** [comment]
The extention is no longer supported

**Comment by informatik01** [comment]
Now it's official. This is what I've just seen in Chrome today:

![Image](https://github.com/user-attachments/assets/bfddc63e-ec5c-425b-b1c7-7af34e7c3d5e)

**Comment by SupianIDz** [comment]
<img width="1387" alt="Image" src="https://github.com/user-attachments/assets/a9e8ca52-1ebf-4286-9177-cc25edca5bdd" />

**Comment by informatik01** [comment]
Yes, we now see this warning, that the extension was turned off in Chrome, but so far it is possible to just _manually reenable_ it and it will still work 🤠.

**Comment by SupianIDz** [comment]
> Yes, we now see this warning, that the extension was turned off in Chrome, but so far it is possible to just _manually reenable_ it and it will still work 🤠.

nah, even though it has been re-enable but still no luck.
<img width="1822" alt="Image" src="https://github.com/user-attachments/assets/fbe2cb42-59f3-45a3-b5d0-75d0a1a25d27" />

**Comment by informatik01** [comment]
@SupianIDz 
You just have to _grant access_ to the extension, if you haven't granted the access to all resources:

![Image](https://github.com/user-attachments/assets/fb17de9d-dbd9-42b9-ac42-54d45fe8fdd7)

![Image](https://github.com/user-attachments/assets/0a0a08e0-dd02-4d50-b35c-5dbf382be3e6)

## Result:

![Image](https://github.com/user-attachments/assets/fe87c327-9667-4400-9d32-565a90eb19b9)

Checked on macOS with the latest version of Chrome at the time of writing this:
> Version 133.0.6943.142 (Official Build) (arm64)

**Comment by justforuse** [comment]
Can we update manifest to V3? is there any plan?

**Comment by oslook** [comment]
This plugin doesn't work anymore, I use this one [JSON Viewer Chrome Extension] below now,
It works pretty well.

https://chromewebstore.google.com/detail/json-viewer-chrome-extens/kggkmmjihpjangbkpcdidlhfbnnfaldn

**Comment by informatik01** [comment]
@oslook 
Actually it still works, you just have to _explicitly_ enable it. Using it now with the latest version of Chrome.

**Comment by justforuse** [comment]
> Can we update manifest to V3? is there any plan?

I just publish a pure V3 version of this, For those who need can check this one [JSON Viewer V3](https://chromewebstore.google.com/detail/json-viewer-v3/cjkilmepecgopoeffjjbhoeenfkponie). Looking forward to feedback and comments

here is the github [repo](https://github.com/justforuse/json-viewer)

**Comment by DigitalCyberSoft** [comment]
> > Can we update manifest to V3? is there any plan?
> 
> I just publish a pure V3 version of this, For those who need can check this one [JSON Viewer V3](https://chromewebstore.google.com/detail/json-viewer-v3/cjkilmepecgopoeffjjbhoeenfkponie). Looking forward to feedback and comments
> 
> here is the github [repo](https://github.com/justforuse/json-viewer)

You haven't pushed any updates to the repository you forked?

Due to security risks associated with this kind of extension, I'd recommend everyone builds it themselves. I've added a fork here that includes the V3 changes and a few fixes https://github.com/DigitalCyberSoft/json-viewer

An old copy of the Chrome Extension changed hands some years back, which led to some malicious abuse of the extension. Hijacking redirects, cookie stuffing, and credential theft.

**Comment by justforuse** [comment]
I pushed to a new branch: https://github.com/justforuse/json-viewer/tree/dev-v3

**Comment by russellvt** [comment]
Looks like Chrome finally force-disabled this one, for me:

> Version 138.0.7204.101 (Official Build) (64-bit)

And despite other forks, above, nothing appears to be published to the Chrome Store (which is "concerning").
